lib/pf/UnifiedApi.pm

```diff
@@ -704,6 +704,8 @@ sub setup_api_v1_config_bases_routes {
         "api.v1.Config.Bases"
     );
 
+    $self->add_subroutes($collection_route, "Config::Bases", "POST", qw(test_smtp fix_permission));
```

Comment:
fix_permission doesn't belong in the base endpoint since it applies changes to many configuration files

Probably better suited in /config/ directly

Reply:
I will put both checkup and fix_permission config the /config endpoint